src/js/components/timelinePage.js

```diff
@@ -1,15 +1,63 @@
 import React from 'react';
 import Timeline from './timeline';
+import PostTweet from './PostTweet';
 
 export default class TimelinePage extends React.Component {
+    constructor(props){
+        super(props);
+        this.state = {
+            activeTab: 'general'
+        }
+    }
+
+    get TABS(){
```

Comment:
does this work? Looks like you're missing a space between `get` and `TABS`
Also it looks like this could be an object stored in a variable instead of a function.

Reply:
since 'const' doesn't work inside of react components, this is the way I found to define readonly objects.

Reply:
`const` totally works inside react but you have to put it in the right place. In an earlier PR I recommended you define your variables within the `render()` function. You could do that here too.

Reply:
^ did you try this?